### ISSUE TITLE
Looking for later version

### ISSUE THREAD

**Issue by j4m13c0** [mineralos/mineralos#1]
Hi, 
Thank you so much for providing this, I was wondering, do you happen to have a later version of this with Inno_a10 support? 
Thank you, 
Jamie

**Comment by jameshilliard** [comment]
I don't think so.

**Comment by j4m13c0** [comment]
> I don't think so.

That's a shame, thank you for the great upload I hope ill be able to do it from this one.

**Comment by jameshilliard** [comment]
Inno firmware should be based on this, they should have the version with Inno_a10 support.

**Comment by j4m13c0** [comment]
> Inno firmware should be based on this, they should have the version with Inno_a10 support.

Yeah, they are based on this, its hard to get support from them though, I will send them a message to see. 
Thanks again.
